pkg/new-ui/v1beta1/frontend/src/app/services/experiment-form.service.ts

```diff
@@ -0,0 +1,412 @@
+import {
+  FormBuilder,
+  Validators,
+  FormGroup,
+  FormArray,
+  FormControl,
+} from '@angular/forms';
+import { Injectable } from '@angular/core';
+import { ObjectiveTypeEnum } from '../enumerations/objective-type.enum';
+import { AlgorithmsEnum } from '../enumerations/algorithms.enum';
+import { BehaviorSubject } from 'rxjs';
+import { createParameterGroup, createNasOperationGroup } from '../shared/utils';
+import { CollectorKind } from '../pages/experiment-creation/metrics-collector/types';
+import { K8sObject, SnackBarService, SnackType } from 'kubeflow';
+import { dump, load } from 'js-yaml';
+import { ObjectiveSpec, AlgorithmSpec } from '../models/experiment.k8s.model';
+
+@Injectable()
+export class ExperimentFormService {
+  constructor(private builder: FormBuilder, private snack: SnackBarService) {}
+
+  /*
+   * helpers for generating default values
+   */
+  defaultObjective(): ObjectiveSpec {
```

Comment:
Do we need this function since we use this one: https://github.com/kubeflow/katib/blob/1b48a137ac24428d83bd411bc1ce4d3f4724d1c6/pkg/new-ui/v1beta1/frontend/src/app/services/experiment-form.service.ts#L53-L63 ?

Reply:
Nope, will remove it